v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,22 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
+SELECT sub.question_id, avg(sub.mean_score_per_user_per_question) as mean_score_per_question, sub.number, corr(sub.mean_score_per_user_per_question, sub2.average_assessment_score_for_user) as discrimination
+FROM (
+    select user_id, question_id, assessment_questions.number, avg(instance_questions.points/assessment_questions.max_points) as mean_score_per_user_per_question
+    from instance_questions
+    JOIN assessment_instances ON instance_questions.assessment_instance_id=assessment_instances.id
+    JOIN assessment_questions ON instance_questions.assessment_question_id=assessment_questions.id
+    WHERE assessment_questions.assessment_id=$assessment_id
+    GROUP BY user_id, question_id, assessment_questions.number
+) sub
+JOIN (
+    select user_id, avg(score_perc)/100 as average_assessment_score_for_user
```

Comment:
Why bother doing the `/100` here? Instead we could just compute `average_assessment_score_perc_for_user`?